app/models/xccdf/benchmark.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+# Stores information about xccdf benchmarks
+module Xccdf
+  class Benchmark < ApplicationRecord
+    has_many :profiles, dependent: :destroy
+    has_many :rules, dependent: :destroy
+    validates :ref_id, uniqueness: { scope: %i[version] }, presence: true
```

Comment:
These are shared by all accounts, right?

Reply:
Yes, that's right.